certified-connectors/IN-D-Payables/apiDefinition.swagger.json

```diff
@@ -0,0 +1,483 @@
+                          },
+                          "conf_score": {
+                            "type": "number",
+                            "format": "float",
+                            "description": "confidence score for extracted po number from invoice",
+                            "title": "confidence score po number"
+                          }
+                        },
+                        "description": "po_number"
+                      },
+                      "vendor_name": {
+                        "type": "object",
+                        "properties": {
+                          "value": {
+                            "type": "string",
+                            "description": "vendor name extracted from invoice",
+                            "title": "vendor name value"
+                          },
+                          "conf_score": {
+                            "type": "number",
+                            "format": "float",
+                            "description": "confidence score for extracted vendor name from invoice",
+                            "title": "confidence score vendor name"
+                          }
+                        },
+                        "description": "vendor_name"
+                      },
+                      "vendor_address": {
+                        "type": "object",
+                        "properties": {
+                          "value": {
+                            "type": "string",
+                            "description": "vendor address extracted from invoice",
+                            "title": "vendor address value"
+                          },
+                          "conf_score": {
+                            "type": "number",
+                            "format": "float",
+                            "description": "confidence score for extracted vendor address from invoice",
+                            "title": "confidence score vendor address"
+                          }
+                        },
+                        "description": "vendor_address"
+                      },
+                      "customer_name": {
+                        "type": "object",
+                        "properties": {
+                          "value": {
+                            "type": "string",
+                            "description": "customer name extracted from invoice",
+                            "title": "customer name value"
+                          },
+                          "conf_score": {
+                            "type": "number",
+                            "format": "float",
+                            "description": "confidence score for extracted customer name from invoice",
+                            "title": "confidence score customer name"
+                          }
+                        },
+                        "description": "customer_name"
```

Comment:
@gayathri35 All the description values are expected to be free from grammatical and spell errors. Please provide valid description values for all the properties.

Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

Reply:
Applies the same comment for all the similar description values in the swagger.

Reply:
@sriyen-msft Sorry for the spell errors, Now I have updated the valid description values for all properties.